rust/bindings_napi/src/lib.rs

```diff
@@ -3,7 +3,6 @@
 use napi::bindgen_prelude::*;
 use napi_derive::napi;
 use parse_ast::parse_ast;
-use xxhash;
```

Comment:
Thanks! Could you also remove the same import here?
https://github.com/rollup/rollup/blob/15d321bf7da5d48ed9a8ed9f87d7f88736ce837d/rust/bindings_wasm/src/lib.rs#L4